### PR TITLE
Redlionfish

### PR DESCRIPTION
**Pull request by perdigao1**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by perdigao1** [comment]
I am willing to be maintainer

**Comment by dopplershift** [comment]
As you continue to make changes to get this recipe working, just push changes to your branch and they'll show up here. Please don't close the PR and make a new one.

**Comment by perdigao1** [comment]
Hi Ryan. Sorry about not dealing with the pull requests correctly
I was attempting to create a new conda-forge package. But it is constantly failing to produce the packages. On my computer I can produce using conda-build, using a similar meta.yaml file.

I am now realising that the problem may be at the github/azure end. I look into details of the "staged-recipes" and, in all of them there is the following error or similar.


[link](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=414472&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=06d62fd2-ab49-5add-9ee5-83b6882d7d2b)


```
Copying D:\a\1\s\recipes\redlionfish\https:\github.com\rosalindfranklininstitute\RedLionfish\archive\refs\tags\v0.3.tar.gz to C:\bld\redlionfish_1637254587965\work\
Traceback (most recent call last):
  File "C:\Miniconda\lib\site-packages\conda_build\utils.py", line 532, in _copy_with_shell_fallback
    subprocess.check_call(f'cp -a {src} {dst}', shell=True,
  File "C:\Miniconda\lib\subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'cp -a D:\a\1\s\recipes\redlionfish\https:\github.com\rosalindfranklininstitute\RedLionfish\archive\refs\tags\v0.3.tar.gz C:\bld\redlionfish_1637254587965\work\v0.3.tar.gz' returned non-zero exit status 1.

```

Any help dealing with this mysterious copy error would be appreciated.

Regards
Luis

File: recipes/redlionfish/meta.yaml
Comment (perdigao1) [reply recipes/redlionfish/meta.yaml]: Done.

1. Changed maintainers name to my github username.
2. Removed comments in meta.yaml and in the script files.

**Comment by perdigao1** [comment]
@conda-forge/staged-recipes
Hi, not sure what happened to this package I thought I had added to conda-forge. It had successfully passed the checks and was available in conda-forge channel last friday, but today it is not available anymore. Any reason?

Second question. If I need to updated (new version), the instructions
[here](https://conda-forge.org/docs/maintainer/updating_pkgs.html)
suggest that the procedure is to clone the "feedstock repository", which I believe should be https://github.com/conda-forge/redlionfish-feedstock . Correct?

File: recipes/redlionfish/build.sh
Comment (dopplershift) [inline recipes/redlionfish/build.sh:1]: I don't understand why a special script is needed and it won't work with the normal `setup.py`? Everything [in there](https://github.com/rosalindfranklininstitute/RedLionfish/blob/main/setup.py) looks completely normal and compatible with conda-forge.

If so you can remove `build.sh` and `bld. bat` and add under `build:` below:
```yaml
build:
  script: "{{ PYTHON }} -m pip install . -vv"
```
which is our standard install line.

**Review by dopplershift — COMMENTED** [review]
I'm not sure what makes you think redlionfish was installable from conda-forge last Friday, but until this PR is merged no feedstock is created (and thus no conda-forge packages are created/uploaded).

File: .gitignore
Comment (dopplershift) [inline .gitignore:19]: Let's not do this, this .gitignore is used by every recipe ever submitted, but this recipe directory will be removed when it's converted to a feedstock.

Comment (perdigao1) [reply .gitignore:19]: oh dear. I can't get it right. First time trying to create a conda-forge package...

**Comment by perdigao1** [comment]
Ok. I made changes as advised, such as removed the build script files.

I have also made changes to the source code, and is now at version 0.5.

File: .gitignore
Comment (dopplershift) [reply .gitignore]: No worries! Learning new things is challenging.

File: recipes/redlionfish/meta.yaml
Comment (dopplershift) [inline recipes/redlionfish/meta.yaml:15]: Did you intend to leave this line like this?

Comment (perdigao1) [reply recipes/redlionfish/meta.yaml:15]: yes. the setup.py is slightly different from the setup_for_condabuild.py file, in the sense that the latter has less requirements. These requirements are included in the meta.yaml file. setup.py is for PyPI or local installations. I had some issues when I tried to have a single file, reason why I made a different one for the conda based installation.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**
